microprofile/WFLY-19592_MicroProfile_Fault_Tolerance_4_1.adoc

```diff
@@ -0,0 +1,127 @@
+---
+categories:
+- microprofile
+stability-level: default
+issue: https://github.com/wildfly/wildfly-proposals/issues/626
+feature-team:
+ developer: rhusar
+ sme:
+  - rhusar
+ outside-perspective:
+  -
+promotes:
+promoted-by:
+---
+= WFLY-19592 [MP Platform 7] MicroProfile Fault Tolerance 4.1
+:author:            Radoslav Husar
+:email:             rhusar@redhat.com
+:toc:               left
+:icons:             font
+:idprefix:
+:idseparator:       -
+
+== Overview
+
+Goal is to integrate MicroProfile (MP) Fault Tolerance (FT) 4.1 specification from the MP Platform 7.0 to WildFly at default stability level.
+To certify functionality, the MP FT TCK is run as part of the WildFly testsuite.
+
+=== User Stories
+
+User wants to deploy an application using MicroProfile Fault Tolerance 4.1 from MicroProfile Platform 7.0.
+
+== Issue Metadata
+
+* https://issues.redhat.com/browse/WFLY-19592[WFLY-19592 [MP Platform 7] MicroProfile Fault Tolerance 4.1]
+
+=== Related Issues
+
+* https://issues.redhat.com/browse/WFLY-19592[WFLY-19588 Implement MicroProfile Platform 7] – MP Platform parent issue
+* https://issues.redhat.com/browse/WFLY-19590[WFLY-19590 MicroProfile Telemetry 2.0] – dependency of this issue for full Telemetry metrics integration
+
+// EAP Tracking issue:
+// * https://issues.redhat.com/browse/EAP7-2225[EAP7-2225 [MP Platform 7] MicroProfile Fault Tolerance 4.1]
+
+=== Affected Projects or Components
```

Comment:
Are the following projects _affected by_ the WildFly feature? I would say it's quite the opposite. Looking at the design template instructions, they go as:

```
<List the projects or components that are affected by the feature. List them using their Git repositories.>
```

Usually what I see here - and I think that's right in terms of usefulness for who will use, test or further integrate the feature - to know what projects will be directly or indirectly affected by the upgrade. Usually WildFly itself is directly affected, i.e. by changing the source code, or documentation etc., while layered projects are indirectly affected. In this very case, I'd say WildFly, WildFly BOMs and Quickstarts projects would be affected. WDYT?

Reply:
Well, the project was adapted to be working in WF so yes, it was affected. But no problem, moved below.